utils/order-utils/src/orders.js

```diff
@@ -288,6 +288,18 @@ const isValidOrder = order => {
   )
 }
 
+function lowerCaseAddresses(order) {
+  for (var key in order) {
+    if (typeof order[key] === 'object') {
+      lowerCaseAddresses(order[key])
+    }
+    if (typeof order[key] === 'string' && order[key].indexOf('0x') === 0) {
```

Comment:
This line will catch `kind`, `version`, `r` and `s` as well as addresses. I imagine that's fine, but just checking you have noticed that?

Reply:
Yes.